webui/react/src/utils/string.ts

```diff
@@ -18,3 +18,8 @@ export const toHtmlId = (str: string): string => {
     .replace(/[^a-z0-9-]/gi, '')
     .toLowerCase();
 };
+
+// prepare an optional string to be used when concatnating strings.
+export const optionalStr = (str?: string, suffix='', prefix=''): string => {
```

Comment:
seems a little overkill to me, but I'll trust your judgement more than mine on what functions would be useful in frontend world

Reply:
yeah, I'm torn here also. the idea is to save some characters or having to define a separate variable outside where you're building a string. especially when the inputs have long variable names :thinking: 
the performance hit of additional fn call wouldn't be concerning here.

Reply:
suggestion:
```
export const listToStr = (list: string[], glue = ' '): string => {
  return list.filter(item => !!item).join(glue);
};
```